<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="AreaDesenho" name="areaDesenho">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>19</y>
      <width>611</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">white</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Sunken</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="botaoFechar">
    <property name="geometry">
     <rect>
      <x>635</x>
      <y>520</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: lightgray;
color: black;
</string>
    </property>
    <property name="text">
     <string>Fechar</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkPonto">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Mostrar pontos</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkTriangulo">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>60</y>
      <width>151</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Mostrar triangulos</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkReta">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>100</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Mostrar retas</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboObjetos">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>150</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="sliderTransX">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>220</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-50</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TickPosition::NoTicks</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sliderTransY">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>250</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-50</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TickPosition::NoTicks</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>190</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Translação</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>948</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Arquivo">
    <property name="title">
     <string>&amp;Arquivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionSalvar"/>
   </widget>
   <widget class="QMenu" name="menu_Informa_es">
    <property name="title">
     <string>Detalhes</string>
    </property>
    <addaction name="actionSobre"/>
   </widget>
   <addaction name="menu_Arquivo"/>
   <addaction name="menu_Informa_es"/>
  </widget>
  <action name="actionAbrir">
   <property name="text">
    <string>&amp;Abrir</string>
   </property>
  </action>
  <action name="actionSalvar">
   <property name="text">
    <string>&amp;Salvar e sair</string>
   </property>
  </action>
  <action name="actionSobre">
   <property name="text">
    <string>&amp;Sobre</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AreaDesenho</class>
   <extends>QFrame</extends>
   <header>areadesenho.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>botaoFechar</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>716</x>
     <y>561</y>
    </hint>
    <hint type="destinationlabel">
     <x>709</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
